src/Renderer/ThreeExtended/OBB.js

```diff
@@ -175,41 +173,55 @@ OBB.extentToOBB = function extentToOBB(extent, minHeight = 0, maxHeight = 0) {
     tmp.cardinals[7]._values[0] = phiStart;
     tmp.cardinals[7]._values[1] = thetaStart + bboxDimension.y * 0.5;
 
-    var cardin3DPlane = [];
+    const cardinalsXYZ = [];
+    const centersLongitude = tmp.cardinals[8].longitude(UNIT.RADIAN);
+    for (var i = 0; i < tmp.cardinals.length; i++) {
+        cardinalsXYZ.push(tmp.cardinals[i].as('EPSG:4978').xyz());
+    }
+
+    return this.cardinalsXYZToOBB(cardinalsXYZ, centersLongitude, true, minHeight, maxHeight);
+};
 
+OBB.cardinalsXYZToOBB = function cardinalsXYZToOBB(cardinals, centerPhi, isEllipsoid, minHeight = 0, maxHeight = 0) {
     tmp.maxV.set(-1000, -1000, -1000);
     tmp.minV.set(1000, 1000, 1000);
-    var halfMaxHeight = 0;
+
+    let halfMaxHeight = 0;
+    tmp.normal.copy(cardinals[8]).normalize();
     tmp.tangentPlaneAtOrigin.set(tmp.normal, 0);
 
     tmp.planeZ.setFromUnitVectors(tmp.normal, tmp.zUp);
     tmp.qRotY.setFromAxisAngle(
-        new THREE.Vector3(0, 0, 1), -tmp.cardinals[8].longitude(UNIT.RADIAN));
+        new THREE.Vector3(0, 0, 1), -centerPhi);
```

Comment:
Why it's necessary to have `centerPhi` in parameter, we could assume that it is in the `cardinals[8]`

Reply:
Because `cardinals` are `xyz` coordinates, not angular.

Reply:
ok
it's rotation on z axis

Reply:
it's even a rotation around the normal of the tile
I will change to rotating `centerPhi`  to `rotationOnNormal`

Reply:
I changed for `centerLongitude` because that's what it is from the user of the function point of view.

Reply:
`centerLongitude` has less meaning in planar.
`qRotY` is incorrect because it's a rotation on Z.
this rotation is done on the normal of the tile, so I will change the names in this direction.
`centerPhi` to `rotationOnNormal` 
`qRotY` to `quaternionOnNormal`

Reply:
> centerLongitude has less meaning in planar.

Doesn't matter since this method is not meant to be used in planar mode.

>  qRotY is incorrect because it's a rotation on Z.

I'll fix this.

>  centerPhi to rotationOnNormal

No. This is not an internal variable but a function parameter name.
And `rotationOnNormal` doesn't make sense for the users of this function - while tile's longitude is quite easy to understand.

>  qRotY to quaternionOnNormal

I don't think repeating the type of the variable in the var name is useful.
I can change to `rotationOnNormal` if you want even if I don't think this name is useful.
I'd prefer something describing the intent: ie what does this rotation means?
For instance if the point of this rotation is to align x/y axes of the tile to the world's X/Y axes, then I'd name it "alignLocalXYAxesToWorldAxes".

Reply:
> `alignLocalXYAxesToWorldAxes`

the axes of the tiles are already aligned. Against this rotation on the normal allows to align the plane of symmetry of the tile on the Y axis.

may be `alignPlaneSymmetryToYAxis`

* ok for `centerLongitude`

Reply:
`align the plane of symmetry of the tile on the Y axis` is unclear. 
Do you meant "to align the axis of symetry of the tile on the Y axis?"
(with the axis of symetry being the line going through cardinals 1-5?)
If it's not what you meant, could you make a simple drawing?

Reply:
To align orthogonal symmetric plane of the tile's geometry on the y axis.
there isn't axis of symmetry  of the tile.